src/authenticationservices.cs

```diff
@@ -1325,9 +1392,193 @@ interface ASAuthorizationSecurityKeyPublicKeyCredentialRegistration : ASAuthoriz
+		string ServerNonceClaimName { get; set; }
+
+		[Export ("customNonceRequestValues", ArgumentSemantic.Copy)]
+		NSUrlQueryItem[] CustomNonceRequestValues { get; set; }
+
+		[Export ("setCustomAssertionRequestHeaderClaims:returningError:")]
+		bool SetCustomAssertionRequestHeaderClaims (NSDictionary<NSString, NSObject> claims, [NullAllowed] out NSError error);
+
+		[Export ("setCustomAssertionRequestBodyClaims:returningError:")]
+		bool SetCustomAssertionRequestBodyClaims (NSDictionary<NSString, NSObject> claims, [NullAllowed] out NSError error);
+
+		[Export ("additionalScopes")]
+		string AdditionalScopes { get; set; }
+
+		[Export ("includePreviousRefreshTokenInLoginRequest")]
+		bool IncludePreviousRefreshTokenInLoginRequest { get; set; }
+
+		[Export ("previousRefreshTokenClaimName")]
+		string PreviousRefreshTokenClaimName { get; set; }
+
+		[Export ("customLoginRequestValues", ArgumentSemantic.Copy)]
+		NSUrlQueryItem[] CustomLoginRequestValues { get; set; }
+
+		[Export ("setCustomLoginRequestHeaderClaims:returningError:")]
+		bool SetCustomLoginRequestHeaderClaims (NSDictionary<NSString, NSObject> claims, [NullAllowed] out NSError error);
+
+		[Export ("setCustomLoginRequestBodyClaims:returningError:")]
+		bool SetCustomLoginRequestBodyClaims (NSDictionary<NSString, NSObject> claims, [NullAllowed] out NSError error);
+
+		[Export ("kerberosTicketMappings", ArgumentSemantic.Copy)]
+		ASAuthorizationProviderExtensionKerberosMapping[] KerberosTicketMappings { get; set; }
+	}
+
+	[NoWatch, NoTV, NoiOS, NoMacCatalyst, Mac (13,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface ASAuthorizationProviderExtensionLoginManager
+	{
+		[Export ("deviceRegistered")]
+		bool DeviceRegistered { [Bind ("isDeviceRegistered")] get; }
+
+		[Export ("userRegistered")]
+		bool UserRegistered { [Bind ("isUserRegistered")] get; }
+
+		[NullAllowed, Export ("registrationToken")]
+		string RegistrationToken { get; }
+
+		[NullAllowed, Export ("loginUserName")]
+		string LoginUserName { get; set; }
+
+		[NullAllowed, Export ("ssoTokens", ArgumentSemantic.Copy)]
+		NSDictionary SsoTokens { get; set; }
+
+		[NullAllowed, Export ("loginConfiguration", ArgumentSemantic.Copy)]
+		ASAuthorizationProviderExtensionLoginConfiguration LoginConfiguration { get; }
+
+		[Export ("saveLoginConfiguration:error:")]
+		bool Save (ASAuthorizationProviderExtensionLoginConfiguration loginConfiguration, [NullAllowed] out NSError error);
+
+		[Protected]
+		[Export ("saveCertificate:keyType:")]
+		void _Save (IntPtr certificate, ASAuthorizationProviderExtensionKeyType keyType);
+
+		[Wrap ("_Save (certificate.GetHandle (), keyType)")]
+		void Save (SecCertificate certificate, ASAuthorizationProviderExtensionKeyType keyType);
+
+		[Protected]
+		[Export ("copyKeyForKeyType:")]
+		[return: NullAllowed]
+		IntPtr _CopyKey (ASAuthorizationProviderExtensionKeyType keyType);
+
+		[return: Release]
+		[Wrap ("new SecKey (_CopyKey (keyType), false);")]
```

Comment:
Instead you need to do this:

```suggestion
		[Wrap ("new SecKey (_CopyKey (keyType), true);")]
```